vipps-ecom-api.md

```diff
@@ -224,90 +221,86 @@ For example, if the vendor's name is "Acme AS" and the vendor offers two differe
 one for point of sale (POS) integrations and one for web shops,
 the headers should be:
 
-| Header | Description | Example value for POS | Example value for webshop | Example value for Vending machines |
-| ------ | ----------- | --------------------- | ------------------------- | ---------------------------------- |
-| `Merchant-Serial-Number`   | The MSN for the sale unit | `123456` | `123456` | `123456` |
-| `Vipps-System-Name`        | The name of the ecommerce solution | `acme` | `acme` | `acme` |
-| `Vipps-System-Version`     | The version number of the ecommerce solution | `1.7` | `2.6` | `2.6` |
-| `Vipps-System-Plugin-Name` | The name of the ecommerce plugin | `acme-pos` | `acme-webshop` | `acme-vending` |
-| `Vipps-System-Plugin-Version` | The version number of the ecommerce plugin   | `3.2` | `4.3` | `4.3` |
+| Header                        | Description                                  | Example value for POS | Example value for webshop | Example value for Vending machines |
+|:------------------------------|:---------------------------------------------|:----------------------|:--------------------------|:-|
+| `Merchant-Serial-Number`      | The MSN for the sale unit                    | `123456`              | `123456`                  | `123456` |
+| `Vipps-System-Name`           | The name of the ecommerce solution           | `acme`                | `acme`                    | `acme` |
+| `Vipps-System-Version`        | The version number of the ecommerce solution | `1.7`                 | `2.6`                     | `2.6` |
+| `Vipps-System-Plugin-Name`    | The name of the ecommerce plugin             | `acme-pos`            | `acme-webshop`            | `acme-vending` |
+| `Vipps-System-Plugin-Version` | The version number of the ecommerce plugin   | `3.2`                 | `4.3`                     | `4.3` |
 
 ## Initiate
 
-Vipps eCommerce API offers two types of payments:
-
-1. Regular eCommerce payments
-2. Express checkout payments
-
-Examples from a demo website:
-
-![Regular and express checkout](images/vipps-flow-web.png)
-
-### Regular eCommerce payments
-
 Payment amounts must be in NOK, be non-zero *and* larger than 1 NOK (1 NOK = 100 øre).
 
 Amounts are specified in minor units.
 For Norwegian kroner (NOK) that means 1 kr = 100 øre.
 Example: 499 kr = 49900 øre.
 
-When you initiate a payment, it will only be *reserved* until you capture it.
-Vipps supports both *reserve capture* and *direct capture* payment flows.
+When you initiate a payment, it will normally only be *reserved* until you capture it.
 
-**Reserve capture** is the default. When you initiate a payment, it will be
-reserved until you capture it.
+This has some benefits:
+* If a payment has been _reserved_, the merchant can
+  make a `/cancel` call to immediately release the reservation and make available
```

Comment:
"make available in the customer's account." sounds clunky.

Reply:
Fixed, thanks!